src/arrayMethodSort.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultCompareFunc = (a, b) => (a + '') > (b + '');
+
+  [].__proto__.sort2 = function(compareFunction = defaultCompareFunc) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.slice(0, i).length; j++) {
+        if (compareFunction(this[j], this[i]) === true
+        || compareFunction(this[j], this[i]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]];
```

Comment:
let's split it with variables

Reply:
@SerhiiKirik I'm not sure I understand you, can you explain?

Reply:
I think that Sergey meant that these values need to be put into variables to make the code more readable. But you shortened the condition, and now in my opinion everything looks normal
![image](https://github.com/mate-academy/js_array-method-sort/assets/54936044/750a2eae-46f0-42a7-beb4-ee8a2b913a77)